packages/transaction-manager/specs/encryption.md

```diff
@@ -50,20 +50,20 @@ The encrypted data, the encrypted keys and a hash of the data are pushed on chai
 
 | Property             | Type   | Description                                                                                                                      |
 | -------------------- | ------ | -------------------------------------------------------------------------------------------------------------------------------- |
-| **data**             | String | First encrypted data of the channel in base64                                                                                    |
+| **encryptedData**             | String | First encrypted data of the channel in base64                                                                                    |
 | **hash**             | String | Normalized Keccak256 hash of the message before encryption                                                                       |
 | **keys**             | Object | AES-256 key encrypted with ECIES from the parties public keys, encoded in base64 and indexed by their multi-formatted identities |
 | **encryptionMethod** | String | Encryption method use for the channel _('ECIES-AES256' here)_                                                                    |
 
-The data are encrypted with the algorithm AES-256 from the channel key generated for this channel.
+The data is encrypted with the algorithm AES-256 from the channel key generated for this channel.
```

Comment:
Both seems to be grammatically correct

Reply:
data is theoretically plural... In practice both work though, so I'm ok with leaving it as is or changing it.